### DISCUSSION TITLE
Block [ ctrl/cmd+W ] too

### DISCUSSION THREAD

**Discussion by maanemann** [Sheeplet1/obsidian-pin-enhancer#3] [Ideas]
Hi! What i hoped this plugin would do was to block ctrl/cmd+W hotkey to close pinned tabs. So if you're still wokring on this and agree that this would be a good addition, consider the idea.

Please tell me, if you implement this. I will uninstall for now, as i never use middle click to close tabs.

Thanks for developing this plugin either way :)

**Comment by Sheeplet1** [comment]
need to rebind the close tab hotkey to the plugin's one
